docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,187 @@
+        second_factor: otp|u2f|webauthn|on
+```
+</TabItem>
+<TabItem label="Teleport Cloud" scope={["cloud"]}>
+Create the following `cluster_auth_preference` dynamic resource:
+
+```yaml
+kind: cluster_auth_preference
+version: v2
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: otp|u2f|webauthn|on
+```
+
+Remember to create your dynamic resource using `tctl create -f <path to your YAML file>`.
+</TabItem>
+</Tabs> 
+
+To make MFA mandatory for all users, `second_factor` must be set to one of the following values:
+
+- `otp`
+- `u2f`
+- `webauthn`
+- `on`
+
+Choose `on` if you would like to require MFA for all users while letting them choose an OTP, U2F, or WebAuthn device. The other options restrict users to a single type of MFA device, which is useful for enforcing a particular standard of security. Once you start the Teleport Proxy Service with the `second_factor` configuration option set to one of these values, Teleport will mandate MFA by:
+
+- Adjusting the Teleport signup page so a user must enroll an MFA device of the kind you have selected. If the value of `second_factor` is `on`, users will have the option to select from multiple device types.
+- Presenting the user with an MFA challenge when they run `tsh login`.
+
+For authentication connectors besides `local`, you must make MFA mandatory via your identity provider, e.g., by adjusting your settings within GitHub or Okta.
+
+
+## Present an MFA challenge for every attempt to access a resource
+After a user logs into a Teleport cluster, they can request access to a particular resource, e.g., a node, database, application, or Kubernetes cluster. In this case, the Teleport Auth Service issues a single-use certificate for accessing that resource. You can prevent attackers from doing damage with a compromised certificate by enabling per-session MFA. With this setting, whenever a user requests a one-time certificate to access a resource, the Teleport Auth Service will issue an MFA challenge, even if the user has already begun a Teleport session via `tsh login`.
+
+To enable per-session MFA for all users, do the following depending on your Teleport environment:
+
+<Tabs>
+<TabItem label="Self-hosted" scope={["oss","enterprise"]}>
+Make the following changes to your Teleport configuration file:
+
+```yaml
+auth_service:
+    authentication:
+        require_session_mfa: yes
+```
+
+</TabItem>
+<TabItem label="Teleport Cloud" scope={["cloud"]}>
+Create the following `cluster_auth_preference` dynamic resource:
+
+```yaml
+kind: cluster_auth_preference
+version: v2
+metadata:
+  name: cluster-auth-preference
+spec:
+  require_session_mfa: yes
+```
+
+Remember to create your dynamic resource using `tctl create -f <path to your YAML file>`.
+</TabItem>
+</Tabs> 
+
+The `require_session_mfa` setting only applies to users who have already established a Teleport session. To require MFA for `tsh login`, see the [previous section](#make-mfa-mandatory-for-tsh-login).
```

Comment:
> The `require_session_mfa` setting only applies to users who have already established a Teleport session.

I'm not sure what this is referring to. Maybe rephrase somehow?

Reply:
This sentence restates information that the page introduces earlier, so I think it should be fine to remove it.